app/src/main/java/edu/umd/cmsc436/mstestsuite/MainActivity.java

```diff
@@ -185,6 +185,24 @@ public void run() {
         });
     }
 
+    public void sendFeedbackToDoc() {
+        setContentView(R.layout.feedback);
```

Comment:
This method should start a new activity, not change the content of the existing one, because now it's very difficult to go back.  Also it should say `@Override` above it, like the other overridden methods.